File: 혜원/week1/code_10818.cpp
#include <iostream>

using namespace std;

int main() {
	int ind = 0;
	int min = -1000000;
	int max = 1000000;

	cin >> ind;
	int arr[1000001];

	for (int i = 0; i < ind; i++) {
		cin >> arr[i];
		if (max < arr[i]) max = arr[i];
		if (min > arr[i]) min = arr[i];
	}
	cout << min << ' ' << max;

	return 0;

}

//배열의 크기가 너무 쓸데없이 커지는거 같아서 다른 사람들은 어떻게 했나
//찾아봤는데 int arr[ind]; 이런식으로 배열 크기를 변수로 선언해서 했다.
//근데 나는 왜 안되는걸까리

File: 민균/7주차/code_10818.java
package baekjoon.study.week7;

import java.util.Scanner;

public class code_10818 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int arr[] = new int[n];

        for(int i = 0; i<arr.length; i++){
            arr[i] = sc.nextInt();
        }
        int max = arr[0];
        int min = arr[0];

        for(int i = 0; i<arr.length; i++){
            if(arr[i]>max){
                max = arr[i];
            }else if(arr[i]<min){
                min = arr[i];
            }
        }
        System.out.println(min+" "+max);
    }
}
